### DISCUSSION TITLE
Footer page

### DISCUSSION THREAD

**Discussion by QusaiImayyah** [QuestPDF/QuestPDF#426] [Q&A] [answered]
How to view footer only lastpage ?

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. To show a footer only on the last page, you can use `ShowIf` to conditionally render content based on the current page number and total pages.

Should you encounter any further problems, please don't hesitate to open a new discussion with additional context.

Thank you!
